data/human/hails.txt

```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
 		"uranium or uranium" "uranium"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
-
+phrase "friendly civilian"
+    word
+        "Be careful of"
+        "A warning about"
+        "Be suspicious of"
+        "Don't trust"
+    word
+        " "
+    word
+        "those space casinos. I blew"
+    word
+        " "
+    word
+        "a"
+        "two"
+        "three"
+        "four"
+        "five"
+    word
+        " "
+    word
+        "million credits on"
+    word
+        "Bourne"
+        "Spica"
+        "Rust"
+        "Starcross"
+        "Skymoot"
+        "Thunder"
+        "Zug"
+        "Flood"
+        "Longjump"
+        "Glaze"
+        "Deep"
+        "Clink"
+        "Winter"
+        "Solace"
+        "Twinstar"
+        "Sundrinker"
+        "Trinket"
+        "Dancer"
+    word
+        " "
+    word
+        "blackjack."
+        "poker."
+        "backgammon."
+        "baccarat."
+        "roulette."
+phrase "friendly civilian"
+    word
+        "Have you read the latest edition of"
+    word
+        " "
+    word
+        "Cosmic Chronicles"
+        "Stellar Spectacles"
+        "Galactic Gazette"
+        "Quantum Quandaries"
+        "Nebula News"
+        "Phantom Planets"
+        "Vague Voyages"
+        "Dark Dimensions"
+        "Mysterious Moons"
+        "Silent Satellites"
+        "Zesty Zeniths"
+        "Daring Discoveries"
+        "Fiery Frontiers"
+        "Global Galaxies"
+        "Rapid Rockets"
+        "Orbital Overload"
```

Comment:
```suggestion
        "Orbital Overload"
        "Stars and Starships"
```
Might as well include one already mentioned in-game!

(Do wish these would be more than just alliterative, e.g. "Southern Rim Spacer News", "Tarazed Today".....)